index.css

```
@@ -64,97 +84,91 @@ div.base-frame {
   font-family: 'Jua', sans-serif;
 }
 
-.logo {
-  grid-column: 1;
-  align-self: center;
+.page-seperator {
+  background-color: rgb(120, 37, 2);
+  height: 15px;
 }
 
-.logo img {
-  width: 250px;
-  padding-left: 32px;
-}
-
-.image-section {
-  grid-column: 1 / span 3;
-  height: 450px;
-  background-image: 
-    linear-gradient(90deg, rgba(0,0,0,1) 0%, rgba(0,0,0,1) 33%, rgba(245,255,0,0.5) 100%),
-    url('background.jpg');
-  background-blend-mode: overlay;
-  background-position: left;
-  background-size: cover;
-  padding: 0px;
-  margin: 0px;
+.middle-content {
+  background-color: rgb(156, 195, 200);
+  min-height: 100vh;
 }
 
-.image-cta {
-  margin: 0px 50px;
-  padding: 64px;
-  width: 25%;
-  background-color: white;
-  border-radius: 8px;
-  font-family: sans-serif;
+.middle-content-flex-container {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  background-color: #bee1e1;
+  margin: 100px;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
 }
 
-.image-cta button {
+.flex-row {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
 }
 
-.image-cta h1 {
+.text-box {
+  background-color: rgb(219, 219, 219);
+  height: 200px;
+  width: 300px;
+  padding: 10px;
+  margin: 50px;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  border-radius: 4px;
+  font-family: 'Jua', sans-serif;
 }
 
-.image-section-body {
-  padding-top: 50px;
+.page-text-box {
+  font-family: 'Jua', sans-serif;
+  height: auto;
+  width: 80%;
+  align-self: center;
 }
 
-h1 {
-	color: black;
+.cute-image {
+  padding: 10px;
+  margin: 50px;
+  align-self: center;
 }
 
-.link-card-section {
-  width: 740px;
-  margin: 0 auto;
+.cute-image img {
+  width: 200px;
 }
 
-.navbar-header {
-  padding-bottom: 64px;
+h1 {
+	color: hwb(0 0% 100%);
 }
 
 .bottom-content {
-  grid-column: 1 / span 3;
-  display: grid;
-  grid-gap: 32px;
-  grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
-  grid-template-rows: 100%;
-}
-
-.blurb-base {
-  background-color: lightgrey;
-  border-radius: 4px;
-  margin: 32px;
-  padding: 12px;
-  font-family: sans-serif;
-
-  display: grid;
-  grid-template-rows: 1fr 4fr;
-  grid-gap: 32px;
+  background-color: rgb(0, 33, 43);
+  display: flex;
+  justify-content: space-between;
+  min-height: 200px;
+  height: 100%;
 }
 
-.blurb1 {
-  grid-column: 2;
+.footer-text {
+  color: white;
+  justify-self: left;
+  align-self: center;
 }
 
-.blurb2 {
-  grid-column: 3;
-}
-.blurb3 {
-  grid-column: 4;
+.daycare-schedule, td, th {
+  border: 1px solid black;
+  border-collapse: collapse;
+  padding: 5px;
+  
 }
 
-.blurb-title {
-  grid-row: 1;
-  font-size: 14pt;
+.daycare-schedule {
+  align-self: center;
+  justify-self: center;
+  margin-bottom: 5px;
+  width: 100%;
 }
 
-.blurb-body {
-  grid-row: 2;
-}
+.calendar {
+  align-self: center;
```

```suggestion
.calendar {
  width: 100%
  align-self: center;
```
I don't want to get too much into the design since that'll be something Susan has a lot of input on and might be likely to change, but this and the embed tag should be set to take up all of the available space if we're going to stick with the PDF embed.  It's super busy and the default makes the contents unreadable if it's squished into a small section in the middle of the page.

---

#3  Looking into whether we want to stick with pdf for now, or make a calendar ourselves with JS.